test/e2e/cypress/integration/profiles/verification/verify/errors.spec.ts

```diff
@@ -138,12 +140,15 @@ context("Account Verification Error", () => {
             })
           })
 
-          it("unable to verify non-existent account", async () => {
-            cy.get('input[name="email"]').type(gen.identity().email)
+          it("unable to verify non-existent account", () => {
```

Comment:
`async` prevented this test from being executed properly. So it never actually tested what it was supposed to.

I checked, and this seemed to be the only instance of an `async` test body.